autoPyTorch/pipeline/components/preprocessing/tabular_preprocessing/encoding/__init__.py

```diff
@@ -86,15 +87,18 @@ def get_hyperparameter_search_space(self,
                                  "choices in {} got {}".format(self.__class__.__name__,
                                                                available_preprocessors,
                                                                choice_hyperparameter.value_range))
-            if len(choice_hyperparameter) == 0:
```

Comment:
what was the case before that `choice_hyperparameter` was equal to 0? Since if there was a value range it should have not been 0?

Reply:
yeah, I don't know why it was done this way. But it should be checking for the length of categorical columns